MDL Docs, fixed typo, added short video on similar process (#192)
R2Northstar/ModdingDocs: docs/source/guides/tools/MDLModding.rst

While creating an entire Model from scratch is possible it would be an extremely long and tedious task. Instead its recommended to use an existing Model as your base.

> 

Author: whoopsies. seem to have hit space bar on accident lol, just removed it :)

Harmony VPK
-----------

> Harmony VPK is a tool written by the Harmony Team. Its an Electron based GUI for VPKTool. It is used to extract files from VPK files.

Reviewer: ```suggestion
Harmony VPK is a tool written by the Harmony Team. It's an electron-based GUI for VPKTool, and is used to extract files from VPK files.
```

Author: sounds good.

``Crowbar ERROR: The model compiler, "<filename>", does not exist.``

``Crowbar ERROR: The game's "<filename>" file does not exist.``

>     Mostly happens if you did not properly set up Crowbar, make sure you followed the steps in `Setup Crowbar </guides/tools/mdlmodding.html#setup-crowbar-one-time-only>`__.

Reviewer 2: Okay the diff here looks weird on GitHub but it's just addint a missing newline

Author: yeah, dont quite know what happened there, i guess its because its the last line in the file and github is funky but idk.
